src/depends/libDatabase/LevelDB.cpp

```diff
@@ -427,7 +427,7 @@ bool LevelDB::RefreshDB()
 
     leveldb::DB* db;
 
-    leveldb::Status status = leveldb::DB::Open(options, "./" + PERSISTENCE_PATH + "/" + this->m_dbName, &db);
+    leveldb::Status status = leveldb::DB::Open(options, STORAGE_PATH + PERSISTENCE_PATH + "/" + this->m_dbName, &db);
```

Comment:
wouldn't STORAGE_PATH + PERSISTENCE_PATH evaluate to `.//persistence`

Reply:
Yes.. but doesn’t matter.

Reply:
I think need to fix a convention atleast, put it in only one.

Reply:
Intentionally have it that ways. Storage path is any path user want to have.
And persistence path is more of constant if we may want to change in future.

Reply:
i see no need of `/` in the storage path. Either way cannot have it in both.

Reply:
Storage path is also accepted from user via launch.sh.
They can give with one ending with / or without /
And should work in either case. So better to have PERSISTENCE_PATH starting with /

Reply:
OK sure, then can the `STORAGE_PATH` be without the `\` ?

Reply:
I see.. you want to set <STORAGE_PATH> to be . 
Not sure existing function SetSeedNodeHistoricalDB still will work in that case.

Reply:
I will test it later.

Reply:
The best solution is to have the custom paths either always end or start with a `/` but never both. As long as that can be made uniform then a change like this will work without adding extra `/`s to any of the paths.